tests/test_run_release.py

```diff
@@ -22,3 +26,14 @@ def test_invalid_extract_github_owner() -> None:
         "https://example.com",
     ):
         run_release.extract_github_owner("https://example.com")
+
+
+def test_check_magic_number() -> None:
+    db = {
+        "release": Tag("3.13.0rc1"),
```

Comment:
We could also test that non-RC/non-final does not raise.

Reply:
It's a little annoying because it asks a question so we'd have to mock that out. I'll mark that as "future improvement".